graphql/fetchTotalUserBadgesCount.ts

```diff
@@ -2,10 +2,10 @@ import { gql } from "@apollo/client";
 
 export const FETCH_TOTAL_USER_BADGES_COUNT = gql`
```

Comment:
i think you could actually just use the `coding_badges_aggregate` endpoint to get the total available badges

right now you're querying from the user-level table and using distinct to get the unique number of badgeIds that exist.. this isn't wrong but using distinct is a more expensive operation.. in our case there probably isn't a noticeable performance hit but imagine if we had 100s of thousands of badges or something.

instead you could just count the total number of rows in the coding_badges table... the primary key in this table is just a single badge, so something like this query should do the same thing:

```
query MyQuery {
  coding_badges_aggregate {
    aggregate {
      count
    }
  }
}
```

Reply:
thank you for pointing that out.  that makes a lot of sense in terms of efficiency. I really appreciate thinking this way all the time!